src/textfield.vue

```diff
@@ -92,6 +94,16 @@ export default {
   methods: {
     fireInputEvent: function (event) {
       this.$emit('input', event.target.value)
+    },
+    focus () {
+      this.$refs.focusTarget.focus()
+    },
+    focusAndSelect () {
```

Comment:
Yes, I think you said there were lots of things I'd need to know you haven't shared yet - hence the :). I also don't know Jade, so happy to leave it for now. If/when you move the file for textfields into /views, I'm happy to add the extra methods once it's nice plain HTML.